main/src/libraries/wrdac/src/subsystems/subSystem_KARMA.cpp

```diff
@@ -18,6 +18,58 @@ void wysiwyd::wrdac::SubSystem_KARMA::appendDouble(yarp::os::Bottle &b, const do
     sub.addDouble(v);
 }
 
+void wysiwyd::wrdac::SubSystem_KARMA::appendString(yarp::os::Bottle &b, const std::string &str)
```

Comment:
There is no need for this method, this can be done inline. You don't save a single line of code by introducing this method.